devtools/x-core/src/git.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::errors::*;
-use determinator::Paths0;
+use determinator::Utf8Paths0;
```

Comment:
why is Utf8Paths0 used in some places and Utf8Path in others?

Reply:
`Utf8Paths0` stands for a list of null-separated UTF-8 paths, such as produced by `git ls-files -z`. `Utf8Path` is a single path.